ExampleMod/Content/Tiles/ExampleTorch.cs

```diff
@@ -69,9 +69,7 @@ public class ExampleTorch : ModTile
 			AddMapEntry(new Color(200, 200, 200), Language.GetText("ItemName.Torch"));
 
 			// Assets
-			if (!Main.dedServ) {
-				flameTexture = ModContent.Request<Texture2D>("ExampleMod/Content/Tiles/ExampleTorch_Flame");
-			}
+			flameTexture = Mod.Assets.Request<Texture2D>("Content/Tiles/ExampleTorch_Flame");
```

Comment:
`TextureAssets.ItemFlame` exists

Reply:
This is a tile, there is a TextureAssets.Flames used for tiles, but they have a unique index and just complicate things. `Main.tileFlame` docs say not to use them.